About.html

```diff
@@ -0,0 +1,106 @@
+                </ul>
+            </nav>
+        </section>
+    </header>
+    
+    <main>
+        <section class="main-sec1 SS1">
+            <div class="house"><a href="./index.html" class="home-link"><img src="./images/house-silhouette-home-buildings-houses-home-silhouette-15.png" alt="">></a></div>
+            <div class="jj">
+                <h1 class="txt-img">Creative Commons Global Summit 2023</h1>
+                <hr class="lign">
+                <p class="dd">A community of generative art creation using machine learning techniques brings together artists, developers, and
+                researchers who are passionate about exploring the intersection of art and technology. Using machine learning algorithms
+                and neural networks, these creators are able to generate unique and complex art forms that are impossible to create by
+                hand. By leveraging the power of artificial intelligence, they are able to create dynamic and ever-changing pieces of
+                art that challenge our traditional notions of what art can be. This community is constantly pushing the boundaries of
+                what is possible with machine learning, and through collaboration and experimentation, they are creating a new form of
+                art that is both innovative and exciting.</p>
+                <p class="susu">Please contact us per Email for any further questions about CC Global Summit 2023!</p>
+                <a href="#" class="email-link">ccsummit2023@ainove.org ></a>
+            </div>
+        </section>
+
+        <section class="creative-ss">
+            <h3>Main Program</h3>
+            <hr class="lign">
+            <p>This logo of CC Global Summit 2015 was decided throught the logo competion in 2016. <br> After online public voting and Selection Committee's judging process this logo won the prize. <br> After consultation with the head Comittee we optimized the price-winning-logo for this year's summit as following.</p>
+            <img src="./images/cannewlogo2018.png" alt="">
+
+        </section>
+
+        <section class="previous-cc">
+            <h3>See the past CC Global Summits</h3>
+            <hr class="lign">
+            <p>Take a look at the last two CC Global Summits wich took place in California-US.</p>
+            <div class="cc-img">
+                <div class="orangeimg1">
+                    <h2>2017</h2>
+                    <p>CC Global Summit in New York</p>
+                </div>
+                <div class="orangeimg2">
+                    <h2>2013</h2>
+                    <P>CC Global Summit in California</P>
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <footer class="foot">
+        <div class="footer-slide2">
+            <img src="./images/cannewlogo2018.png" alt="logo">
+            <p>&copy; 2023 Generative Art Creation Using Machine Learning Techniques. Some Rights Reserved.</p>
+        </div>
+    </footer>
```

Comment:
- [ ] You claimed to have followed the original design on this section. But that claim does not appear to true. Your design is still missing the partner section on the about page. Please take a look :point_down: 

### Your Design
![your-about-footer](https://user-images.githubusercontent.com/26597432/227609235-aafaa36e-f692-4e9e-858f-31829e393030.png)

### Figma Design
![temp-about-footer](https://user-images.githubusercontent.com/26597432/227609277-6b991b0a-6896-426e-9de0-b6729e038f81.png)

You can see from the screenshots that you desing does not carry the parner part of the footer. Kindly add it to make your design complete on that part of the project.